#! /usr/bin/env python

# Initialize ros and import parameter generator
PACKAGE='surgical_simulator'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Add parameters
#       Name     Type    Level Description     	Default 	Min   	Max
gen.add("xOffset", double_t,0,"X offset from trocar for right robot target [m]",0,-0.4,0.4)
gen.add("yOffset", double_t,0,"Y offset from trocar for right robot target [m]",0,-0.4,0.4)
gen.add("zOffset", double_t,0,"Z offset from trocar for right robot target [m]",0,-0.4,0.4)

exit(gen.generate(PACKAGE, "surgical_simulator", "publishTrocarFrame_params"))